src/utils/css-utils.js

```diff
@@ -5,6 +5,7 @@ const sizes = {
     desktop: 1200,
     tablet: 768,
     phone: 599,
+    verySmall: 280,
```

Comment:
I think we've got a mistake in sizes and media-queries.
As I understand our sizes should be like `phone <= 599 < table <= 768 < desktop <= 1200 < hd`.
So, `min-width` in media query doesn't fit our needs.
I think we should rewrite `media` function to use `max-width` and set `hd` to `min-width`. Also we should rewrite sync functions below too.

@kitos what do you think about this?